core/accounting/src/fiscal_year/repo.rs

```diff
@@ -0,0 +1,28 @@
+use es_entity::*;
+use sqlx::PgPool;
+
+use super::{
+    entity::{FiscalYear, FiscalYearEvent},
+    error::FiscalYearError,
+};
+use crate::primitives::{ChartId, FiscalYearId};
+
+#[derive(EsRepo, Clone)]
+#[es_repo(
+    entity = "FiscalYear",
+    err = "FiscalYearError",
+    columns(
+        chart_id(ty = "ChartId", list_for, parent),
```

Comment:
Can keep probably `update(persist = false)` since we don't want the `chart_id` on an entity to change after it's initialized. This is generally something we do for any entity that references another entity by id.

```suggestion
        chart_id(ty = "ChartId", update(persist = false), list_for, parent),
```